crawl-ref/docs/options_guide.txt

```diff
@@ -829,6 +829,15 @@ explore_greedy = true
         otherwise identical to regular explore. Explore greed is
         disabled if autopickup is off (Ctrl-A).
 
+explore_greedy_visit = (pile | enchanted | both)
```

Comment:
Use stacks instead of piles to match the other `explore_stop options`

Reply:
updated to use stacks

Reply:
In case you are going to be testing this, adding 
`dprf("Options.explore_greedy_visit %i stack_greed: %i, glowing_greed:%i, artefact_greed:%i", Options.explore_greedy_visit, stack_greed, glowing_greed, artefact_greed);`
to line 302 in stash.cc is pretty good for seeing that it works.